vignettes/using-summary-log.Rmd

```diff
@@ -0,0 +1,156 @@
+```
+
+```{r setup, include = FALSE}
+#set_model_directory("../tests/testthat/model-examples-complex")
+
+# set babylon execution path
+if (Sys.which("bbi") == "") {
+  options('rbabylon.bbi_exe_path' = read_bbi_path())
+}
+```
+
+# What's in a Model Summary?
+
+As mentioned above, the `bbi_summary_log_df` tibble contains a lot of information, which is a subset of what is contained in the `bbi_nonmem_summary` object returned from `model_summary()`
+
+```{r model summary single}
+MODEL_DIR <- "../tests/testthat/model-examples-complex"
+
+sum1 <- "iovmm" %>%
+          read_model(.directory = MODEL_DIR) %>%
+          model_summary()
+
+names(sum1)
+```
+
+For example, the `run_details` section alone contains wealth of information about this model run:
+
+```{r run details single}
+str(sum1$run_details)
+```
+
+Much of this is very useful, but it's also a bit intimidating, and it can take some work to unpack it all and find the bits and pieces you're looking for.
+
+# What's in the Summary Log?
+
+The `summary_log()` function is designed to extract some of the most relevant diagnostics and model outputs from a batch of model summaries and organize them into a more easily digestible tibble. Like `run_log()` and `config_log()`, it takes two arguments:
+
+* `.base_dir` -- Directory to look for models in.
+* `.recurse` -- Logical indicating whether to search recursively in subdirectories. This is `TRUE` by default.
+
+```{r fake summary_log for names}
+sum_df <- summary_log(MODEL_DIR)
+names(sum_df)
+```
+
+```{r real summary_log, include = FALSE}
+# we're using test data that requires a fail flag, but this is not the focus of this vignette so we hide it.
+sum_df <- summary_log(MODEL_DIR, .fail_flags = list(ext_file = "1001.1.TXT"))
+```
+
+The specific columns returned are described below, though there is also a list of them, with brief definitions, in the [`summary_log()` docs](https://metrumresearchgroup.github.io/rbabylon/reference/summary_log.html) that can be accessed any time with `?summary_log()` in the console.
+
+## Housekeeping Columns
+
+The first column is `absolute_model_path` which contains an absolute path that unambiguously identifies each model. This serves as the primary key for the tibble. 
+
+The second column contains the `bbi_nonmem_summary` object, discussed above, for each model. This can be extracted and manipulated if you would like more detailed data from it.
+
+The `error_msg` and `needed_fail_flags` columns describe whether `bbi` had any trouble parsing the model outputs. These won't be discussed in detail here. Refer to the [`summary_log()` docs](https://metrumresearchgroup.github.io/rbabylon/reference/summary_log.html) for more information.
+
+## Run Details Columns
+
+The next batch of columns contain the core diagnostics and model outputs. 
+
+* `ofv` -- Objective function value _with no constant_ from the final estimation method. The constant, and the value _with_ the constant can be found in `bbi_nonmem_summary$ofv`
```

Comment:
If we include these here, then we need to keep it in sync with the roxygen documentation for `summary_log()`. I think it would be okay to say something like

> refer to the documentation for `summary_log()` for a description of the columns

Reply:
Yea, that's fair. I'll change it to just reference the docs.